cli/types/cypress.d.ts

```diff
@@ -2876,13 +2876,13 @@ declare namespace Cypress {
      * Override default config options for Component Testing runner.
      * @default {}
      */
-    component: Omit<ResolvedConfigOptions, TestingType>
+    component: ComponentConfigOptions<ComponentDevServerOpts>
```

Comment:
Why did we remove the `Omit` here?

Reply:
`ResolvedConfigOptions` contains `component` and `e2e` members.
They are not in `ComponentConfigOptions` because they are already omitted in the constructor `CoreConfigOptions`. No need to Omit them anymore.